code/modules/mob/living/carbon/superior_animal/roach/types/toxic.dm

```diff
@@ -20,6 +20,11 @@
 	acceptableTargetDistance = 5
 	kept_distance = 3
 
+	// Armor related variables
+	bullet = 20
+	bio = 75
+	rad = 50
```

Comment:
```suggestion
	rad = 100
```
Gestrahltes are radioactive and spit radioactive goo, if they were not immune they would have already died.